scripts/asm-installer/install_asm

```diff
@@ -234,6 +241,7 @@ EOF
     start_managed_control_plane
     configure_package
     install_managed_components
+    grant_managed_logging_permissions
```

Comment:
This does not respect the lack of `--enable-gcp-iam-roles` option

```suggestion
    if can_modify_gcp_iam_roles; then
    grant_managed_logging_permissions
    fi
```
probably a warning should be also given to indicate that users should add the permission themselves